Documentation/conf.py

```diff
@@ -77,9 +77,12 @@
 if branch is None or branch == 'latest':
     branch = 'HEAD'
     archive_name = 'master'
+    chart_release = './cilium'
 elif branch == 'stable':
     branch = release
     archive_name = release
+    chart_release = 'cilium/cilium --version ' + release
```

Comment:
`chart_release` doesn't seem to be defined for the `else` condition below, I think it still needs to be added with the same contents as this line.

Reply:
The `tags` logic would also be necessary below.

Reply:
Updated